<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionOuvrir"/>
    <addaction name="actionSauvegarder"/>
    <addaction name="actionFermer"/>
    <addaction name="separator"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menu_propos">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="action_propos_de_Qt"/>
    <addaction name="action_propos"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menu_propos"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOuvrir">
   <property name="text">
    <string>Ouvrir...</string>
   </property>
  </action>
  <action name="actionFermer">
   <property name="text">
    <string>Fermer</string>
   </property>
  </action>
  <action name="actionSauvegarder">
   <property name="text">
    <string>Sauvegarder</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="action_propos_de_Qt">
   <property name="text">
    <string>À propos de Qt...</string>
   </property>
  </action>
  <action name="action_propos">
   <property name="text">
    <string>À propos...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
